pygtfs/gtfs_entities.py

```diff
@@ -70,6 +70,8 @@ def in_range(self, key, value):
 def _validate_float_range(float_min, float_max, *field_names):
     @validates(*field_names)
     def in_range(self, key, value):
+        if value is None:
```

Comment:
Doesn't this make all of the `in_range` validated fields nullable? What am I missing?

Reply:
Since some floats are nullable, this rule was previously too strict, but you're right- this solution may be overly broad. 

I can look into adding a `nullable` argument so this check won't fail only for nullable columns.

Reply:
Please check this, I think I've made the necessary change.